### PR TITLE
Update Rollout Scorer to create new scorecards in arcade

### PR DESCRIPTION
**Pull request by adiaaida**
Fixes https://github.com/dotnet/core-eng/issues/15718.

### CODE REVIEW

**Review by missymessa — APPROVED** [review]
LGTM shipit

File: src/RolloutScorer/RolloutScorer/StandardConfig.cs [hunk 1]
```diff
@@ -58,7 +58,7 @@ public static class StandardConfig
             GithubConfig = new GithubConfig
             {
                 ScorecardsGithubOrg = "dotnet",
-                ScorecardsGithubRepo = "core-eng",
+                ScorecardsGithubRepo = "arcade",
                 ScorecardsDirectoryPath = "Documentation/Rollout-Scorecards/",
```
Comment (jonfortescue) [inline src/RolloutScorer/RolloutScorer/StandardConfig.cs:62]: Is it all right to leave it on the same path or is there a different place we want to put it? (Maybe near the dev guide?)

Comment (adiaaida) [reply src/RolloutScorer/RolloutScorer/StandardConfig.cs:62]: Oh good idea. I'll move them to the TeamProcess directory. Thoughts on migrating the old cards to arcade?

Comment (missymessa) [reply src/RolloutScorer/RolloutScorer/StandardConfig.cs:62]: Let's migrate them. Even if the links in the OneNote break, we'll still have them all together.

Comment (adiaaida) [reply src/RolloutScorer/RolloutScorer/StandardConfig.cs]: PR incoming.

**Review by jonfortescue — APPROVED** [review]

**Pull request merged**